k8s/demo/common-overlay/fees-pay/kustomization.yaml

```diff
@@ -20,6 +20,7 @@ patchesStrategicMerge:
 - ../../../namespaces/fees-pay/ccpay-bubble-frontend/demo.yaml
 - ../../../namespaces/fees-pay/ccpay-paymentoutcome-web/demo.yaml
 - ../../../namespaces/fees-pay/ccpay-payment-api/demo.yaml
+- ../../../namespaces/fees-pay/ccpay-callback-function/demo.yaml
```

Comment:
Can you remove this as part?

Reply:
I have removed it now, had not seen the initial existing entry.